com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Removed
 
 ### Fixed
-
+- Fixed `NetworkSceneManager` does not synchronize despawned in-scene placed NetworkObjects nor do they have to register them to re-spawn them. (#1898)
```

Comment:
Grammar is a little weird here. Maybe "`NetworkSceneManager` no longer synchronizes..."

Reply:
Actually, I must have been sleepy when I wrote that.  :)
It is a fix and a change.